todo-list/src/TodoContainer.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Todoitem from './toDoItem';
+
+class TodoContainer extends React.Component {
+  render() {
+    return this.props.todos.map((task) => (
+      <Todoitem key={task.id} todos={task} handleCheck={this.props.handleCheck} />
+    ));
+  }
+}
+
+TodoContainer.propTypes = {
+  todos: propTypes.array.isRequired,
+};
+
+export default TodoContainer;
```

Comment:
### You have Implemented events for the checkbox, please implement also those functionalities
- [CSS in Reactjs app](https://ibaslogic.com/css-in-reactjs-app/).
- [edit to dos items](https://ibaslogic.com/how-to-edit-todos-items-in-react/).
- [React lifecycle methods](https://ibaslogic.com/react-lifecycle-methods/).
- [React Router](https://ibaslogic.com/routing-with-react-router/).
### Optional Tasks, However, I strongly recommend you to take them into account as they can make teach you a lot.
- [React state in local storage](https://ibaslogic.com/persisting-react-state-in-local-storage/) (Optional).
- [SVG icons in React](https://ibaslogic.com/how-to-use-svg-icons-in-react-project/) (Optional).
- [hamburger menu in React](https://ibaslogic.com/how-to-add-hamburger-menu-in-react/) (Optional).
- [Deploying React app to GitHub Pages](https://ibaslogic.com/deploying-react-app-to-github-pages/) (Optional).
![Screenshot 2022-09-17 093715](https://user-images.githubusercontent.com/52010789/190844115-5ea8b3c8-6d53-46c8-b4e3-952882976c3f.png)